test/test_hpytype.py

```diff
@@ -1508,7 +1510,7 @@ def test_get_name(self):
         assert mod.Dummy.__name__ == "Dummy"
         assert mod.get_name(mod.Dummy) == "Dummy"
         assert mod.get_name(str) == "str"
-        assert mod.get_name(array.array) == "array"
+        assert mod.get_name(struct.Struct) == "Struct"
```

Comment:
is `array.array` problematic for PyPy?

Reply:
Not so much problematic as awkward. Using it means we need to add another module to the untranslated tests, which slows the testing down ever so slightly (PyPy's untranslated tests build "just enough" of the interpreter to run tests, with a limited set of modules). `struct` is part of the core modules we require always.